doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
I agree that keeping them in sync is not really worth our time if no one is using them, but putting them out on the web with all the cleaned up versions seems like giving them more attention than they really deserve